chore: add KeptnApp migration script (#2959)
keptn/lifecycle-toolkit: docs/docs/migrate/keptnapp/index.md

3. Add the `app.kubernetes.io/managed-by: keptn` annotation
   to the `KeptnApp` resource if it is not already there.

> You can migrate your KeptnApp manually or, if you have go installed

Reviewer: ```suggestion
You can migrate your KeptnApp manually or, if you have
[go](https://go.dev/)
installed
```

on your machine, use the script provided
[here](https://github.com/keptn/lifecycle-toolkit/tree/main/lifecycle-operator/converter).

```bash
    go run convert_app.go path_to_keptnapp_to_convert path_to_desired_output_file
```

> For instance, to run the example file conversion, you can do:

Reviewer: ```suggestion
For instance, to run the example file conversion, the command is:
```
